src/components/dashboard/jumbotron/NullJumbotron.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+/**
+ *  NullJumbotron displays when the user has no upcoming trips
+ */
+
+export default class NullJumbotron extends Component {
```

Comment:
I don't think this needs its own component if it's only going to render two static paragraph elements.